.cbmc-batch/jobs/aws_cryptosdk_edk_list_clean_up/aws_cryptosdk_edk_list_clean_up_harness.c

```diff
@@ -35,9 +35,8 @@ void aws_cryptosdk_edk_list_clean_up_harness() {
     __CPROVER_assume(aws_cryptosdk_edk_list_elements_are_valid(&edk_list));
 
     aws_cryptosdk_edk_list_clean_up(&edk_list);
-    // assert(aws_cryptosdk_edk_list_is_valid(&edk_list));
-    // assert(aws_cryptosdk_edk_list_elements_are_valid(&edk_list));
+    assert(AWS_IS_ZEROED(edk_list));
 }
-
+#include <aws/common/error.inl>
```

Comment:
Why do we need commented includes bellow this line? Why do we need to keep the include at the end of the proof harness (perhaps a comment here)?

Reply:
done